recipes/cpputest/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class CppUTestConan(ConanFile):
+    name = "cpputest"
+    description = \
+"CppUTest is a C /C++ based unit xUnit test framework for unit testing and for test-driving your code." \
+"It is written in C++ but is used in C and C++ projects and frequently" \
+"used in embedded systems but it works for any C/C++ project."
+    license = "BSD-3-Clause"
+    topics = ("conan", "testing", "unit-testing")
+    homepage = "https://cpputest.github.io"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+        "with_extensions": ["ON", "OFF"],
```

Comment:
```suggestion
        "with_extensions": [True, False],
```

Not because ON, OFF are invalid, but because True, False is a pattern well. It's a bit confusing mixing both: `conan instal ... -ocpputest:fPIC=True -ocpputest:with_extension=ON

Reply:
That's a great point - thanks. I'll accept it.